concent_api/core/message_handlers.py

```diff
@@ -989,7 +989,8 @@ def handle_messages_from_database(client_public_key: bytes) -> Union[message.Mes
         elif pending_response.response_type == PendingResponse.ResponseType.ForceReportComputedTaskResponse.name:  # pylint: disable=no-member
             if pending_response.subtask.ack_report_computed_task is not None:
                 ack_report_computed_task = deserialize_message(
-                    pending_response.subtask.ack_report_computed_task.data.tobytes())
+                    pending_response.subtask.ack_report_computed_task.data.tobytes()
+                )
```

Comment:
All the changes in this file seem unrelated to the commit at all.

Reply:
This is one of the small one-line non-related changes that we discussed on latest retro.